agenthub/codeact_agent/codeact_agent.py

```diff
@@ -204,7 +204,7 @@ def step(self, state: State) -> Action:
                 '</execute_bash>',
                 '</execute_browse>',
             ],
-            'temperature': 0.0,
+            'temperature': self.llm.config.temperature,
```

Comment:
On 2nd thought: I think this line should be removed since the `self._completion` partial in LLM class already passes in the temperature?